### ISSUE TITLE
Espanso enables/disables itself automatically many times

### ISSUE THREAD

**Issue by molitona** [federico-terzi/espanso#751]
Hi, I've noticed that espanso enable/disable itself automatically many times. 
I'm not pressing double tab by mistake.
What's the matter ?

> 20:57:25 [ INFO] espanso version 0.7.3
> 20:57:25 [ INFO] using config path: /home/user/.config/espanso
> 20:57:25 [ INFO] using package path: /home/ser/.local/share/espanso/packages
> 20:57:25 [ INFO] spawning worker process...
> 20:57:25 [ INFO] Binded to IPC unix socket: /home/user/.local/share/espanso/espanso.sock
> 20:57:25 [ INFO] initializing worker process...
> 20:57:25 [ INFO] Binded to IPC unix socket: /home/user/.local/share/espanso/worker.sock
> 20:57:26 [ INFO] watching for changes in path: /home/user/.config/espanso
> 20:57:26 [ INFO] Using modulo at "/usr/bin/modulo"
> 20:57:26 [ INFO] worker is running!
> 22:35:06 [ INFO] Toggled: espanso disabled
> 22:35:19 [ INFO] Toggled: espanso enabled
> 23:14:59 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:15:23 [ INFO] Toggled: espanso disabled
> 23:15:23 [ INFO] Toggled: espanso enabled
> 23:16:31 [ INFO] Toggled: espanso disabled
> 23:16:31 [ INFO] Toggled: espanso enabled
> 23:19:33 [ INFO] Toggled: espanso disabled
> 23:19:33 [ INFO] Toggled: espanso enabled
> 23:19:33 [ INFO] Toggled: espanso disabled
> 23:19:33 [ INFO] Toggled: espanso enabled
> 23:19:33 [ INFO] Toggled: espanso disabled
> 23:19:33 [ INFO] Toggled: espanso enabled
> 23:22:31 [ INFO] Toggled: espanso disabled

**Comment by federico-terzi** [comment]
Hey,

I'm sorry you experienced this issue! By default the toggle key is ALT rather than TAB, are you sure you didn't press ALT repeatedly?

You can also customize or disable the toggle key entirely: https://espanso.org/docs/configuration/#customizing-the-toggle-key

Feel free to reopen if this doesn't solve your issue

Cheers :)

**Comment by molitona** [comment]
Yes, I'm 100% sure. See press time interval, so you can conclude yourself.

**Comment by molitona** [comment]
@federico-terzi

**Comment by federico-terzi** [comment]
>  See press time interval, so you can conclude yourself.

I see, thank you. I didn't notice the first time!
I'm wondering if you have a stuck key in your keyboard, as this is the first time I hear about this issue...

Also, could you please try changing the toggle key to another one (such as RIGHT_CTRL) and see if that helps? Perhaps there is some configuration/software on your machine that is interfering with espanso :)

**Comment by molitona** [comment]
I don't have stuck key. My keyboard is perfect. Otherwise, I changed toggle key and I'll tell you later
